packages/flutter/test/painting/star_border_test.dart

```diff
@@ -117,7 +117,7 @@ void main() {
   testWidgets('StarBorder basic geometry', (WidgetTester tester) async {
     await testBorder(tester, 'basic_star', const StarBorder());
     await testBorder(tester, 'basic_polygon', const StarBorder.polygon());
-  });
+  }, skip: true); // TODO(flutter): https://github.com/flutter/flutter/issues/143101.
```

Comment:
Should we disable this only when running form flutter/engine?

This does not affect the framework because the framework uses a different method for testing in presubmit since it is authenticated directly with Gold. 

Running golden file tests from a different repo is more expensive, and likely the root cause.

Reply:
Ok I concur with this. I won't close the PR just yet, but I won't submit it either. 

Let's try tweaking how engine smoke tests are run first.